cmd/opm/generate/cmd.go

```diff
@@ -27,7 +27,8 @@ func NewCmd() *cobra.Command {
 
 func newDockerfileCmd() *cobra.Command {
 	var (
-		baseImage      string
+		binaryImage    string
```

Comment:
You know, I actually prefer `baseImage` because it makes more sense in the context of using "scratch", but the flag name is `binary-image`, so I understand this change.

Another option that crossed my mind (which is more complex code, but better end-user UX):
- Deprecate/hide `--binary-image`
- Add `--base-image` (and let it take over the shortened `-i`)
- Make `--base-image` and `--binary-image` mutually exclusive.

But then I also thought to myself: "But long-term I still want better tooling that doesn't require users to have to do their own docker builds, so maybe we minimize work and complexity here because \[the above suggested changes] don't actually matter all that much."

Reply:
I mean ... me too on the `base-image` thing, because that's what it is.  In this case, it was exposed as `binary-image` so I just aligned things. 
And I had already played with the same kind of thinking w.r.t. mutual-exclusivity and renaming to `base-image` and it's not much code, so let me push up what I think that looks like.

Reply:
```shell
./bin/opm generate dockerfile --binary-image scratch --base-image scratch foo
Flag --binary-image has been deprecated, use --base-image instead
Error: if any flags in the group [binary-image base-image] are set none of the others can be; [base-image binary-image] were all set
Usage:
  opm generate dockerfile <dcRootDir> [flags]
```